### DISCUSSION TITLE
Warning in the Community building and other modelling function

### DISCUSSION THREAD

**Discussion by Mushahid2521** [micom-dev/micom#185] [General]
Hi there,
I hope you are having a great day! Thanks so much for offering Micom. 
I am trying to build metabolic model for a consortia of bacteria. I have used Carveme for generating the single organism metabolic profile (i.e. the .xml) file and then built the reference database using Micom. Now when I run any modelling I get the following warning all the time. 

_WARNING:cobra.medium.boundary_types:Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this._

Is this something I should worry about and what does that mean if you please let me know would be much appreciated. 
Thank you.

**Comment by cdiener** [comment]
It's all good. CARVEME names its external compartment slightly different (`C_e` instead of `e`) so MICOM uses a heuristic to find the external compartment which will work with CARVEME and GAPSEQ. So in this case you can ignore the warning or suppress it in case it is spamming your output.

**Reply by Mushahid2521** [reply]
Thanks @cdiener.
